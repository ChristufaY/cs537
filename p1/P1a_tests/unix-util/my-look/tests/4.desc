Sample non-alphanumeric test: ignore non-alphanumerical chars
