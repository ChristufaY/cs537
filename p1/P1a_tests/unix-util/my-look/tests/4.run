cat /usr/share/dict/words | ./my-look gnus
