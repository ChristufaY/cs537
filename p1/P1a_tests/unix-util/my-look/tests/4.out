GNU's
gnu's
gnus
